server/controllers/companyController.js

```diff
@@ -0,0 +1,85 @@
+const { default: mongoose } = require("mongoose");
+const Company = require('../models/companyModel')
+
+// get all companies
+const getCompanies = async (req, res) => {
+
+    const companies = await Company.find({}).sort({ createdAt: -1 })
```

Comment:
Why do you use find with an empty object? find will return always the first object that matches that pattern so I think you wouldn't even have to call sort afterwards.

Reply:
As I learned that using find with an empty object is required to get all the object. I would try it without them to see the results